### PR TITLE
Katas UI string review - Oracles

### PR DESCRIPTION
**Pull request by SoniaLopezBravo**

### CODE REVIEW

File: katas/content/oracles/index.md [hunk 1]
```diff
@@ -60,9 +60,9 @@ where $\ket{x_i}$ represents the state of the $i$-th qubit.
 Oracles must be unitary transformations, and follow the same rules of linear algebra as other quantum operations.
 This allows us to define quantum oracles based on their effect on the basis states - tensor products of single-qubit basis states $\ket{0}$ and $\ket{1}$.
 
-> For example, an oracle that implements a function that takes two bits of input will be defined using its effect on basis states $\ket{00}$, $\ket{01}$, $\ket{10}$, and $\ket{11}$.  
+> For example, an oracle that implements a function that takes two bits of input is defined using its effect on basis states $\ket{00}$, $\ket{01}$, $\ket{10}$, and $\ket{11}$.  
 
-There are two types of quantum oracles: phase oracles and marking oracles.  Let's take a closer look at them.
+There're two types of quantum oracles: phase oracles and marking oracles.  Let's take a closer look at them.
```
Comment (billti) [inline katas/content/oracles/index.md:65]: I didn't even know this was a valid contraction. I think it reads far worse than "There are", personally.

**Review by billti — APPROVED** [review]

File: katas/content/oracles/index.md [hunk 2]
```diff
@@ -16,23 +16,23 @@ Quantum oracles are a key part of many quantum algorithms that rely on quantum i
...
+Some classical problems (typically <a href="https://en.wikipedia.org/wiki/Decision_problem" target="_blank">decision problems</a>) are also expressed in terms of oracles; in this case the implementation of the function isn't important, but the functionality that it provides.  
 
-> Suppose I provided you a function which takes two list parameters as input, where these lists represent the availability of two employees at a company during the week.  The function returns true if there is a day (Monday, Tuesday, Wednesday, Thursday, or Friday) for which they are both free and could schedule a meeting, and false if no such date exists.
+> Suppose you've given a function which takes two list parameters as input, where these lists represent the availability of two employees at a company during the week.  The function returns true if there is a day (Monday, Tuesday, Wednesday, Thursday, or Friday) for which they are both free and could schedule a meeting, and false if no such date exists.
```
Comment (cesarzc) [inline katas/content/oracles/index.md:35]: Would this be better phrased as "you've been given" or "you're given"?

**Review by cesarzc — APPROVED** [review]

File: katas/content/oracles/index.md (hunk 2 above)
Comment (billti) [inline katas/content/oracles/index.md:35]:
```suggestion
> Suppose you've been given a function which takes two list parameters as input, where these lists represent the availability of two employees at a company during the week.  The function returns true if there is a day (Monday, Tuesday, Wednesday, Thursday, or Friday) for which they are both free and could schedule a meeting, and false if no such date exists.
```

File: katas/content/oracles/index.md (hunk 1 above)
Comment (billti) [inline katas/content/oracles/index.md:65]:
```suggestion
There are two types of quantum oracles: phase oracles and marking oracles.  Let's take a closer look at them.
```

**Pull request merged**